lib/menu.rb

```diff
@@ -0,0 +1,29 @@
+class Menu
+  def initialize
+    @dishes = [
```

Comment:
I wonder if storing the prices as floats rather than strings would be preferable. I can see below you then have to convert these into numbers to get the total cost of the order. If this program were to get more complicated it might be more flexible if these were stored as floats.